client/me/purchases/remove-purchase/index.jsx

```diff
@@ -312,8 +314,25 @@ class RemovePurchase extends Component {
 		);
 	}
 
+	renderMarketplaceSubscriptionsDialog() {
+		const { purchase, activeSubscriptions } = this.props;
+		return (
+			<MarketPlaceSubscriptionsDialog
+				isDialogVisible={ this.state.isDialogVisible }
+				isRemoving={ this.state.isRemoving }
+				closeDialog={ this.closeDialog }
+				removePlan={ this.removePurchase }
```

Comment:
Instead of directly removing the plan purchase we should render the initial plan dialog so that we don't introduce regressions

https://github.com/Automattic/wp-calypso/blob/96e744b73923836a8fccda12419833bd2a1a1b3d/client/me/purchases/remove-purchase/index.jsx#L245-L259